tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -51,6 +51,7 @@
 import roslaunch.core
 import roslaunch.config
 import roslaunch.depends
+from . import cached
```

Comment:
Please avoid relative imports but use the absolute package name instead (PEP 328).

Same below.